packages/cli/src/commands/login/command.ts

```diff
@@ -0,0 +1,49 @@
+import { Command } from '../help';
+import { getPkgName } from '../../util/pkg-name';
+
+export const loginCommand: Command = {
+  name: 'login',
+  description: 'Authenticate a email or team.',
```

Comment:
This doesn't sound accurate.

```suggestion
  description: 'Authenticate using your email or team id.',
```